albam/engines/mtfw/structs/mrl.ksy

```diff
@@ -18,36 +19,131 @@ seq:
 types:
   resource_binding:
     seq:
-      - {id: resource_type, type: u4}
-      - {id: resource_index, type: u4}
-      - {id: unk_01, type: u4}
-
+      - {id: info, type: cmd_info} # value type
+      - id: value_cmd
+        type:
+          switch-on: info.cmd_type
+          cases:
+            0: hash_block
+            1: cmd_ofs_buffer
+            2: hash_block
+            3: cmd_tex_idx
+            4: hash_block
+      - {id: shader_object_hash, type: u4}
+    instances:
+      float_buffer:
+        pos: _parent._io.pos + value_cmd.as<cmd_ofs_buffer>.ofs_float_buff
+        type: 
+          switch-on: shader_object_hash
+          cases:
+            0x7b2c215f: shd_s_globals # $Globals re0
+            0x6c801200: shd_cb_material # CBMaterial re0
+            0x7b2c2159: shd_s_globals # $Globals rehd
+            0x6c8011f9: shd_cb_material # CBMaterial rehd
+            0x7b2c2155: shd_s_globals # $Globals rer1
+            0x6c8011f4: shd_cb_material # CBMaterial rer1
+            0x7b2c215e: shd_s_globals # $Globals rer2
```

Comment:
I thought in all games same commands for increasing the float buffer have the same size but in the case of ReRev2 $Globals size is bigger than 76 floats

Reply:
I'll run tests to verify it. I think we could add an `app_id` parameter to decide on size as well.

Reply:
minimal tests for mrl pushed. Will improve tests later.